src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
```

Comment:
Perhaps rename the file to be just   RunTest.csx
There could be some value in uniformity here. - can just move from one test directory to another and run same command.